buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java

```diff
@@ -92,7 +92,7 @@ public String call() throws Exception {
                 return String.format(
                     "%s/distributions/%s-%s.pom",
                     project.getBuildDir(),
-                    getArchivesBaseName(project),
+                    pomTask.getName().toLowerCase().contains("zip") ? project.getName() : getArchivesBaseName(project),
```

Comment:
Hey @reta just for my knowledge, if any future, if we have any other publish tasks (like tar), do we have to explicitly add to contains block?

Reply:
Hey @prudhvigodithi , we have to distingush artifacts somehow: f.e. JAR and ZIP do have different artifact names. If we try to publish ZIP and TAR under the same artifact, I think it won't work: the POMs will override each other since the only difference between them is packaging - it is not sufficient.